debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -487,6 +488,37 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
                     + " matched against this regular expression. If matched the error is changed to retriable.")
             .withDefault(false);
 
+    public static final Field ERRORS_MAX_RETRIES = Field.create("errors.max.retries")
```

Comment:
Those field definitions should go to `EmbeddedEngine`, as they are not regular connector properties.